scripts/foundry/transmuter/TransmuterPause.s.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import "transmuter/transmuter/Storage.sol" as Storage;
+import { ISettersGuardian } from "transmuter/interfaces/ISetters.sol";
+import { ProxyAdmin } from "oz/proxy/transparent/ProxyAdmin.sol";
+import { Enum } from "safe/Safe.sol";
+import { MultiSend, Utils } from "../Utils.s.sol";
+
+contract PauseTransmuter is Utils {
+    function run() external {
+        address[] memory collateralList = transmuter.getCollateralList();
+        bytes memory transactions;
+        uint8 isDelegateCall = 0;
+        address to = address(transmuter);
+        uint256 value = 0;
+        {
+            bytes memory data = abi.encodeWithSelector(
+                ISettersGuardian.togglePause.selector,
+                address(0x0),
```

Comment:
Maybe can define the address we'll have in the dotenv

Reply:
No this is because we don't need a collateral address for the redeem